src/ApiGenerator/Generator/Razor/RazorGeneratorBase.cs

```diff
@@ -71,32 +69,28 @@ protected async Task DoRazor<TModel>(TModel model, string viewLocation, string t
             Func<TModel, string> identifier, Func<string, string> target,
             CancellationToken token
             )
-        {
-            using (var c = pbar.Spawn(items.Count, "Generating namespaces", new ProgressBarOptions
-            {
-                ProgressCharacter = '─',
-                ForegroundColor = ConsoleColor.Yellow
-            }))
-            {
-                foreach (var item in items)
-                {
-                    var id = identifier(item);
-                    var targetLocation = target(id);
-                    await DoRazor(item, viewLocation, targetLocation, token);
-                    c.Tick($"{Title}: {id}");
-                }
-            }
-        }
+		{
```

Comment:
whitespace changes?

Reply:
It's because of the change from a using block to a using statement, so indent changed.